server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/IMAPServer.java

```diff
@@ -312,4 +312,8 @@ protected ChannelHandlerFactory createFrameHandlerFactory() {
     public Set<ConnectionCheck> getConnectionChecks() {
         return this.connectionChecks;
     }
+
+    public ReactiveThrottler getReactiveThrottler() {
```

Comment:
Does it makes sense to expose the throttler outside?

Shouldn't we only expose if the queue is full?

Reply:
It will be quite complicated for testing if throttler is not exposed. We are still allowed to use Reflection, right?

Reply:
Then :
 - Still add a method to only expose if the queue is full
 - Keep this methods but reduce its visibility: package private and make it clear this is for tests: `@VisibleForTesting` annotation.